### PR TITLE
fix(tor): news/blog links properly update to tor links

### LINKED ISSUE #3542: Trezor News links are not being open as .onion
**Describe the bug**

Trezor News links are not being open as `.onion` addresses.

**Info:**
 - Suite Version: app (AppImage), 21.3.1, 1dbc5b720515c39a47f1dc58391dbebf1d766394
 - OS: NixOS 

**Steps to reproduce:**
1. Go to the tor settings, enable "_Open trezor.io links as .onion links_".
2. Go to the Dashboard.
3. Click on any link on blog.trezor.io.

**Expected result:**
Opens as `.onion` or rename the Settings option to exclude Medium.

**Actual result:**
Opens blog.trezor.io directly.

### PR DESCRIPTION
**Pull request by keraf**
Fixes #3542

### CODE REVIEW

File: packages/suite/src/views/dashboard/components/NewsFeed/index.tsx [hunk 1]
```diff
@@ -107,10 +112,11 @@ const NewsFeed = ({ maxVisibleCount = 9 }: Props) => {
                     <Post
                         key={item.link}
                         target="_blank"
-                        href={item.link}
+                        href={isTor ? toTorUrl(item.link) : item.link}
                         data-test={`@dashboard/news/post/${index}`}
                     >
-                        <Image src={item.thumbnail} />
+                        {console.log(item)}
```
Comment (slowbackspace) [inline packages/suite/src/views/dashboard/components/NewsFeed/index.tsx]: forgotten console.log

Comment (keraf) [reply packages/suite/src/views/dashboard/components/NewsFeed/index.tsx]: Woops :sweat_smile:

Comment (keraf) [reply packages/suite/src/views/dashboard/components/NewsFeed/index.tsx]: Addressed in 81437a3

File: packages/suite/src/views/dashboard/components/NewsFeed/index.tsx [hunk 2]
```diff
@@ -81,11 +83,14 @@ interface Props {
     maxVisibleCount?: number;
 }
 
+const BLOG_URL = 'https://blog.trezor.io/';
```
Comment (prusnak) [inline packages/suite/src/views/dashboard/components/NewsFeed/index.tsx]: Already defined here: https://github.com/trezor/trezor-suite/blob/d3a5d232f105d6ca53e15dbed52848e677eb3771/packages/suite/src/constants/suite/urls.ts#L8

Comment (keraf) [reply packages/suite/src/views/dashboard/components/NewsFeed/index.tsx]: Addressed in latest rebase/squash (44cf70a)

**Review by slowbackspace — APPROVED** [review]
I have a feeling we could use TrezorLink component instead of our own custom `a`, but I would expect some trouble there regarding overriding default Link styling and anyway such refactor can always be part of another PR.

Ci is failing, but probably only due to node version deps problem we keep running into.

disclaimer: I haven't tested it. 😁

**Pull request merged**